src/components/notes/Sidebar.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export default function Sidebar(props) {
```

Comment:
Use es6 and try to destructure the props.
```suggestion
const Sidebar = ({newNote, notes, ...}) => {
```